ui/oopsyraidsy/data/04-sb/dungeon/ala_mhigo.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+// Ala Mhigo
+[{
+  zoneRegex: /Ala Mhigo/,
+  damageWarn: {
+    'Ala Mhigo Magitek Ray': '24CE', // Line AoE, Legion Predator trash, before boss 1
+    'Ala Mhigo Lock On': '2047', // Homing circles, boss 1
+    'Ala Mhigo Tail Laser 1': '2049', // Frontal line AoE, boss 1
+    'Ala Mhigo Tail Laser 2': '204B', // Rear line AoE, boss 1
+    'Ala Mhigo Tail Laser 3': '204C', // Rear line AoE, boss 1
+    'Ala Mhigo Shoulder Cannon': '24D0', // Circle AoE, Legion Avenger trash, before boss 2
+    'Ala Mhigo Cannonfire': '23ED', // Environmental circle AoE, path to boss 2
+    'Ala Mhigo Aetherochemical Grenado': '205A', // Circle AoE, boss 2
+    'Ala Mhigo Integrated Aetheromodulator': '205B', // Ring AoE, boss 2
+    'Ala Mhigo Circle Of Death': '24D4', // Proximity circle AoE, Hexadrone trash, before boss 3
+    'Ala Mhigo Exhaust': '24D3', // Line AoE, Legion Colossus trash, before boss 3
+    'Ala Mhigo Grand Sword': '24D2', // Conal AoE, Legion Colossus trash, before boss 3
+    'Ala Mhigo Art Of The Storm 1': '2066', // Proximity circle AoE, pre-intermission, boss 3
+    'Ala Mhigo Art Of The Storm 2': '2587', // Proximity circle AoE, intermission, boss 3
+    'Ala Mhigo Vein Splitter 1': '24B6', // Proximity circle AoE, primary entity, boss 3
+    'Ala Mhigo Vein Splitter 2': '206C', // Proximity circle AoE, helper entity, boss 3
+    'Ala Mhigo Lightless Spark': '206B', // Conal AoE, boss 3
+  },
+  triggers: [
+    {
+      id: 'Ala Mhigo Demimagicks',
+      damageRegex: '205E',
+      condition: function(e, data) {
+        // Double taps only
+        return e.type != '15';
+      },
+      mistake: function(e, data) {
+        return { type: 'warn', blame: e.targetName, text: e.abilityName };
+      },
+    },
+    {
+      id: 'Ala Mhigo Unmoving Troika',
+      damageRegex: '2060',
+      condition: function(e, data) {
+        // Non-tanks shouldn't be hit
+        return data.role != 'tank';
```

Comment:
This doesn't work.  `data.role` is your role, not the person's role.  Given that party information is now an event when using ngld overlay, this could be hooked up in the future, but not now.

Reply:
Rats, I knew I'd forgotten something. Okay. We have a couple choices on this one. I could just remove it entirely, although I don't like that option. I could comment it with a FIXME for when party information *is* available. (My preferred choice, although I can understand if you prefer not to merge commented code.) Or I could remove the tank condition and simply have all double taps be warnings. I don't like that one as much, but it's better than just removing it. What do you think?

Reply:
I think having this be a double tap is probably correct even if we had party information.  Since if a tank is dead and it's on somebody else (and the tank gets rezzed?), the tank probably still shouldn't be hit by it, ideally? So, I think just warning on double taps seems the best.